README.md

```diff
@@ -1,2 +1,31 @@
 # drupal8-migration-terms
-Module to Migrate Drupal 8 terms to Drupal 8
+
+Module to migrate Drupal8 terms to Drupal 8
+
+Instructions:
+
+- Add credentials to settings.php file:
+
+$databases['migration_drupal8']['default'] = array(
+  'driver' => 'mysql',
+  'database' => 'db_old_drupal8',
+  'username' => 'root',
+  'password' => 'root',
+  'host' => 'mariadb',
+  'port' => '3306',
+  'prefix' => '',
+  'collation' => 'utf8mb4_general_ci',
+);
+
+- Enable 'drupal8_migration_terms' custom module and its dependencies.
+
+$ drush en drupal8_migration_terms -y
```

Comment:
Use Drupal Console instead of Drush and link to drupalconsole project

Reply:
Done